.circleci/terraform/load-test/main.tf

```diff
@@ -7,12 +7,6 @@ provider "aws" {
   assume_role {
     role_arn = var.role_arn
   }
-
-  default_tags {
```

Comment:
It would be good to keep these tags so that if the TF destroy fails, we can go back and clean them up manually. It's also helpful for cost accounting if we ever wanted to look at what this costs to run.

Reply:
I agreed, but the `default_tags` seems not working with this old version of aws provider (see the [error log](https://app.circleci.com/pipelines/gh/hashicorp/consul/32504/workflows/736b3714-1777-4121-b433-37e5bba6149b/jobs/704717?invite=true#step-108-1478)). After removing these lines, the terraform apply works.

There is a `always` step in our circleCI job to destroy the resources:
https://github.com/hashicorp/consul/blob/915f1089f03f25b3ca0bf6edd7c719d0c9b00f4e/.circleci/config.yml#L1012-L1017

Maybe we can re-enable the global tag after we upgrade the aws provider and module. WDYT?

Reply:
Sounds good to me.